parsons/databases/postgres/postgres.py

```diff
@@ -0,0 +1,74 @@
+from parsons.utilities import check_env
+from parsons.databases.postgres.postgres_core import PostgresCore
+import logging
+import os
+
+
+logger = logging.getLogger(__name__)
+
+
+class Postgres(PostgresCore):
+    """
+    A Postgres class to connect to database. Credentials can be passed from a ``.pgpass`` file
+    stored in your home directory, environmental variables or passed arguments.
+
+    Args:
+        username: str
+            Required if env variable ``PGUSERNAME`` not populated
+        password: str
+            Required if env variable ``PGPASSWORD`` not populated
+        host: str
+            Required if env variable ``PGHOST`` not populated
+        db: str
+            Required if env variable ``PGDB`` not populated
```

Comment:
Similar to above this should be `PGDATABASE` instead of `PGDB`

>[PGDATABASE behaves the same as the dbname connection parameter.](https://www.postgresql.org/docs/12/libpq-envars.html)